web-components/src/components/inputs/validation.ts

```diff
@@ -117,3 +118,23 @@ export function validatePrice(
   }
   return;
 }
+
+/**
+ * Checks if the given string is an ETH address
+ *
+ * @method isValidEthAddress
+ * @param {String} address the given HEX adress
+ * @return {Boolean}
+ */
+const isValidEthAddress = (address: string): boolean => {
```

Comment:
Very minimal validation here. I was trying to avoid bringing in a whole library like `ethers` or `web3` if this is all we need.